understanding/20/use-of-color.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Use of Color</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Color is not the only way of distinguishing information</dd>
+		<dt>Author task</dt><dd>Use other things than just color to convey meaning</dd>
```

Comment:
> other things

Maybe better "Use other information such as shape or text in addition to convey meaning"